lib/src/components/NxList/NxListButtonItem.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+import classnames from 'classnames';
+import React, { forwardRef } from 'react';
+import { NxFontAwesomeIcon } from '../..';
+import {NxListButtonItemProps, nxListButtonItemPropTypes} from './types';
+import { includesDisabledClass } from '../../util/classUtil';
+
+const NxListButtonItem = forwardRef<HTMLLIElement, NxListButtonItemProps>(
+    function nxListButtonItem({ children, className, disabled, selected, buttonClassName,
+      buttonAttributes, ...attrs }, ref) {
+      const liClassNames = classnames('nx-list__item', 'nx-list__item--clickable', className);
+      const buttonClassNames = classnames('nx-list__btn', buttonClassName, { selected, disabled });
+      return (
+        <li ref={ref} className={liClassNames} {...attrs}>
```

Comment:
Could use `NxList.Item` here to make things slightly cleaner

Reply:
> Could use NxList.Item here to make things slightly cleaner

Actually nevermind, not if `NxList.Item` is defined using `withClass` as I suggested in another comment; that would remove the ability to have the `ref`, which is probably important here since this is clickable

Reply:
> > Could use NxList.Item here to make things slightly cleaner
> 
> Actually nevermind, not if `NxList.Item` is defined using `withClass` as I suggested in another comment; that would remove the ability to have the `ref`, which is probably important here since this is clickable

NxList.Item is now defined using `withClass`.